test_integration/lib/test/realtime/realtime_auth_url_test.dart

```diff
@@ -0,0 +1,42 @@
+
+import 'dart:io';
+
+import 'package:ably_flutter/ably_flutter.dart';
+import 'package:ably_flutter_integration_test/app_provisioning.dart';
+import 'package:ably_flutter_integration_test/factory/reporter.dart';
+
+//Make sure that creating realtime with authURL won't crash
+Future<Map<String, dynamic>> testCreateRealtimeWithAuthUrl({
+  required Reporter reporter,
+  Map<String, dynamic>? payload,
+}) async {
+  reporter.reportLog('init start');
```

Comment:
can this log be removed?

Reply:
I think it should stay as this is the standard for every test.